src/sagemaker/cli/compatibility/v2/modifiers/serde.py

```diff
@@ -0,0 +1,407 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""Classes to modify serializer and deserializer code to be compatible with
+version 2.0 and later of the SageMaker Python SDK.
+"""
```

Comment:
I guess I've seen Eric quote [PEP 257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) enough times now :joy: - this should be a one-liner

Reply:
Fixed